.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: C
+addons:
+    apt:
+        packages:
+        - libgmp-dev
+        - libmpfr-dev
+install: bash scripts/build_cross_compiler.sh
```

Comment:
The tests don't actually need the cross compiler at all. They just need a C11 capable compiler.